src/ykfde-open

```diff
@@ -137,10 +160,18 @@ if [ "$YKFDE_PRINT_ONLY" ]; then
   exit 0
 fi
 
+if [ "$TEST_PASSPHRASE" ]; then
+  [ "$DBG" ] && printf '%s\n' " > Passing '$YKFDE_PASSPHRASE' to 'cryptsetup'"
+  [ "$DBG" ] && printf '%s\n' " > Decrypting with 'cryptsetup luksOpen $TEST_PASSPHRASE $LUKS_KEYSLOT $YKFDE_LUKS_DEV'..." || echo " > Decrypting with 'cryptsetup'..."
+  printf %s "$YKFDE_PASSPHRASE" | cryptsetup luksOpen "$TEST_PASSPHRASE" "$LUKS_KEYSLOT" "$YKFDE_LUKS_DEV" 2>&1
```

Comment:
When `$LUKS_KEYSLOT` is not set this fails. Please move it as last param: `$TEST_PASSPHRASE"  "$YKFDE_LUKS_DEV" "$LUKS_KEYSLOT"`. Do the some to debug line above for consistency.